python_anticaptcha/base.py

```diff
@@ -39,8 +40,19 @@ def get_captcha_text(self):  # Image
         return self._last_result['solution']['text']
 
     def report_incorrect(self):
-        return self.client.reportIncorrectImage(self.task_id)
+        warnings.warn(
+            "report_incorrect is deprecated, use report_incorrect_image instead",
+            DeprecationWarning
+        )
+        return self.client.report_incorrect_image()
```

Comment:
@joaocns , I added deprecation warning to avoid breaking public interface of library & tools of our users.

Reply:
I agree with you. Would be better if I keep the report_incorrect() reporting images, and add the a new report_incorrect_recaptcha() function, as an extra feature?